ManatiCR/retopais
# RP-3: Problems page, RP-9: Blog landing page

https://manati.atlassian.net/browse/RP-3

How-to-test:
- Go to http://retopais.dev/admin/config/retopais/problematicas and configure as necessary
- Ensure all problems terms have images
- Go to http://retopais.dev/problematicas, it should look like the prototype (https://www.estudiomanati.com/prototipos/retopais/#p=problematicas&c=1)

By error, this PR also contains https://manati.atlassian.net/browse/RP-9

How-to-test:
- Go to http://retopais.dev/admin/config/retopais/blog and configure as necessary
- Create some blog posts in different categories
- Go to /blog, you should have a list of blogs and it should look like https://www.estudiomanati.com/prototipos/retopais/#p=blog&c=1
- Go to a section by clicking a link in the blog page
- That page should look like https://www.estudiomanati.com/prototipos/retopais/#p=blog__seccion_&c=1
